OpenShift/helm/nexus-iq/README.md

```diff
@@ -0,0 +1,50 @@
+# Sonatype IQ server
+
+[Sonatype Nexus IQ Server](https://www.sonatype.com/nexus-iq-server)  is everything you need to know to trust your software supply chain. It powers Nexus Firewall, Nexus Lifecycle, and Nexus Auditor.
+
+## Chart Configuration Options
+
+| Parameter            | Description                                                  | Default           |
+| -------------------- | ------------------------------------------------------------ | ----------------- |
+| `replicaCount`       | Number of IQ Service replicas                                | `1`               |
```

Comment:
I know there are lots of configs we don't necessarily "support" yet, but I wonder if replicaCount is meaningful if 1 is the only acceptable value.

Reply:
I'll strip it, good point.